modules/search/js/search-widget-filters-admin.js

```diff
@@ -1,4 +1,10 @@
-( function( $ ) {
+/* globals jetpack_search_filter_admin, jQuery */
+
+( function( $, args ) {
+	var defaultFilterCount = ( 'undefined' !== typeof args && args.defaultFilterCount ) ?
+		args.defaultFilterCount :
+		10;
```

Comment:
Why this double-default to `10` here ?

Reply:
I added that there as a fallback in case the localization didn't get picked up. I'll update it to 5 :)